### ISSUE TITLE
Compatibility 

### ISSUE THREAD

**Issue by hweckhardt** [cdaecke/md_fullcalendar#4]
Any plans to make this extension compatible with Typo3 10.4.x and calendarize 7... ?

Many thanks in advance.

**Comment by cdaecke** [comment]
I have just release version 2.0.0, which is compatible with TYPO3 10. Since there is no final version 7 of `ext:calendarize`, I did not check the compatibility with that.

**Comment by hweckhardt** [comment]
Hi,

 

Many thanks for your message.

 

I have installed md_fullcalender together with calendarize 7.0.dev, as the calendarize version before had errors in BE Main configuration, which the author corrected with the version 7. I have “hacked” your ext_emconf to enable the installation. 

 

On the first view, it is working – thanks for your extension!

 

Horst-Walter Eckhardt

 

 

Von: chris [mailto:notifications@github.com] 
Gesendet: Mittwoch, 19. August 2020 16:47
An: cdaecke/md_fullcalendar
Cc: hweckhardt; Author
Betreff: Re: [cdaecke/md_fullcalendar] Compatibility (#4)

 

I have just release version 2.0.0, which is compatible with TYPO3 10. Since there is no final version 7 of ext:calendarize, I did not check the compatibility with that.
